release-notes/release-notes58.md

```diff
@@ -25,7 +25,9 @@ keywords: releasenotes bugfixes
 
 The following new features were added for the {{site.productname}} 5.8 release.
 
-### Feature name
+### Improved keyboard navigation for the inline boundary feature
+
+Users can now escape inline elements (such as links and comments) using the `Home`, `End`, `PageUp`, and `PageDown` keys, see: [Content appearance options - Inline boundaries]({{site.baseurl}}/configure/content-appearance/#inline_boundaries).
```

Comment:
Also do we need to mention this isn't applicable to Mac for page up/down? as Macs define a different behaviour for page up/down.